### ISSUE TITLE
config.rasi

### ISSUE THREAD

**Issue by AlvinPix** [adi1090x/rofi#137]
# When running rofi it has problems with the following file config.rasi

```
$ bash config.rasi
```

- config.rasi: línea 1: /bin: Is a directory
- config.rasi: línea 2: applets: order not found
- config.rasi: línea 3: syntax error near unexpected element `('
- config.rasi: línea 3: ` * Author : Aditya Shakya (adi1090x)'

**Comment by ldgnu** [comment]
Deleting the following lines is ok and does not give any more error
Line 53 to 57 of the config.rasi

	/*---------- Fallback Icon ----------*/
	run,drun {
		fallback-icon: "application-x-addon";
	}
	
	rofi  -v
Version: 1.7.1

**Comment by NovusEdge** [comment]
It's a simple typo. Here's what it should be:
```
/*---------- Fallback Icon ----------*/
run-drun {
	fallback-icon: "application-x-addon";
}
```

I'll make a PR for the minor fix if it's this issue's still up for taking

**Comment by NovusEdge** [comment]
I've closed the PR in favor of #147

**Comment by tonmoy998** [comment]
Thank you
